### ISSUE TITLE
Eror

### ISSUE THREAD

**Issue by alvino9** [script3/yieldblox-beta#378]
the first time I opened tesnet I tried to borrow, but the web suddenly shut down and couldn't be used and I accidentally paid my loan with the value Me,
Later when I returned to apply for a loan, there was a sign that I could borrow more than the balance I had, but I didn't do it because I was afraid of getting hit.Liquidity Because there are not enough funds to repay the loan

**Comment by arnob016** [comment]
The beta testing period is already over , if this is your first time sorry bud.

**Comment by alvino9** [comment]
No,that was my village when I first joined Beta tesnet, and because of
that, I only got a little ybx because I lost funds at the beginning

Pada tanggal Sel, 9 Nov 2021 22.58, TrainerADP016 ***@***.***>
menulis:
